packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -41,6 +41,10 @@ typedef InteractiveViewerWidgetBuilder = Widget Function(BuildContext context, Q
 /// robust positioning of an InteractiveViewer child that works for all screen
 /// sizes and child sizes.
 ///
+/// See [Lazy Flutter performance](https://www.youtube.com/watch?v=qax_nOpgz7E)
```

Comment:
Nit: Should this be under "See also" below?

Reply:
Nit: Capitalize "p" in "performance".

Reply:
Will fix, but it's a copy/paste of the video title